### PR TITLE
Went through most of the German entries in "Annoyances - Remains"

### PR DESCRIPTION
**Pull request by DandelionSprout**
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behaviour**: 

I personally understand it as such that the "Remains" category in AdGuard Annoyances isn't really supposed to exist anymore, because such entries are nowadays meant be part of the regular anti-ad lists.

This time around I've actually browsed through the sites in question, one at a time, to see which entries I could reproduce (and move over to AdGuard German Filter), and which entries I couldn't reproduce (and thus were removed entirely).

I also stumbled across 6 new entries to add:
```
! https://www.autobild.de/artikel/audi-rs-4-vs.-rs-6-beschleunigung-0-100-v8-17086377.html
autobild.de##.rectangle
autobild.de##.partnerlabel
! https://www.derwesten.de/
||img.derwesten.de/img/incoming/*-315x80.jpg^
||img.derwesten.de/img/*-140x53.jpg^
||img.derwesten.de/img/incoming/*-315x80-$image
! https://www.tagesspiegel.de/
tagesspiegel.de###urban-teaser1
```

I suppose it deserves note that I had managed to forget that AdGuard had a German list, thus I only had EasyList Germany turned on during the testing... 😅 

[//]: # (Replace %screenshot_url% below with a link to the screenshot of the problem. Also, you can paste image from clipboard instead. It will be automatically loaded.)

* **Expected behaviour**: 

The entries in the "Remains" category are gradually re-reviewed and processed, until eventually all entries have been re-assigned and the "Remains" category can be deleted.

***Steps to reproduce the problem***:

N/A

***System configuration***

**Filters:**

image

Information                            | Value
---                                    | ---
Operating system:                      | Windows 10
Operating system version (Android/iOS) | May 2020 Update
Browser:                               | Chrome x64
AdGuard version:                       | Nano Adblocker 1.0.0.152 + Nano Defender 15.0.0.201
Filters enabled:                       | See screenshot above
AdGuard mode (Android only):           | N/A
Filtering quality (Android only):      | N/A
HTTPS filtering (Android only):        | On
Simplified filters (iOS only)          | Off
AdGuard DNS:                           | None
Stealth mode options (Windows only)    | ?
Helpdesk ID (if exists):               | ?

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

### CODE REVIEW

**Comment by Alex-302** [comment]
@BlazDT please take a look. As I see, some rules are not dangerous and hide visible ad leftovers.

File: AnnoyancesFilter/sections/remains.txt [hunk 1]
```diff
@@ -73,7 +69,6 @@ bursa.com##.bg--gri[style="padding: 10px 0;"]
 m.pornxs.com###im_scroll_close_button
 m.smutty.com##.ui-content .center[style*="margin-"]:not([id])
 m.empflix.com##.bannerBlock
-t-online.de##div[data-dy^="Partner:"]:not([data-dy^="Partner:Telekom"])
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/remains.txt:76]: Rule is found on t-online.de when I check the site, so please add it to German filter, thank you.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: GermanFilter/sections/specific.txt [hunk 2]
```diff
@@ -2123,7 +2123,7 @@ ruhr24.de,wetterauer-zeitung.de,giessener-allgemeine.de,fr.de,wlz-online.de,da-i
 wallstreet-online.de##body > #siteteaser
 wallstreet-online.de##.newsFlagSlider
 wallstreet-online.de###mooTickerContainer
-! https://github.com/AdguardTeam/AdguardFilters/pull/6023*
+! https://github.com/AdguardTeam/AdguardFilters/pull/60233
```
Comment (BlazDT) [inline GermanFilter/sections/specific.txt:2126]: No need to include link to pull request.

File: AnnoyancesFilter/sections/remains.txt (hunk 1 above)
Comment (DandelionSprout) [reply AnnoyancesFilter/sections/remains.txt:76]: I do have the "Allow edits by maintainers" button turned on, you know. But sure thing, I'll take care of it. Give me 2min.

Comment (DandelionSprout) [reply AnnoyancesFilter/sections/remains.txt:76]: Oh, wait, now I remember. `t-online.de##div[data-dy*="Partner:"]` is already in both AdGuard German and in EasyList Germany, thus making the entry superfluous after all.

File: GermanFilter/sections/specific.txt (hunk 2 above)
Comment (DandelionSprout) [reply GermanFilter/sections/specific.txt]: Has been removed now.

**Review by BlazDT — APPROVED** [review]

**Pull request merged**